src/releasers/java-yoshi.ts

```diff
@@ -287,4 +288,46 @@ export class JavaYoshi extends ReleasePR {
     }
     return newVersions;
   }
+
+  // Begin release configuration
+
+  // Override this method to use static branch names
+  // If you modify this, you must ensure that the releaser can parse the tag version
+  // from the pull request.
+  protected async buildBranchName(
```

Comment:
Am I following that each language would opt in to this new functionality? Is the reasoning to allow for a gradual upgrade and testing process?

Reply:
Yes, this allows each releaser to choose its strategy for branch naming and will allow gradual upgrades. This is `protected` so it's not part of the public surface area. If we're happy with using `release-please/branches/[branch]/(components/[component])?`globally, then we can refactor this to be done in the base class without breaking anyone.